app/elements/socobo-recipe/socobo-recipe-overview.html

```diff
@@ -508,7 +508,7 @@
           for (var e in el) {
             var listItem = el[e];
             var internalEl = Polymer.dom(listItem.root).querySelector(".check paper-checkbox");
```

Comment:
I know! But when I tried this with only .isChecked it didn't work for me because isChecked didn't change the visible state of the checkboxes. So I needed to check or uncheck the particular boxes manually.
I tried your code snipped by copy and pasting it, but it still does not work for me. Maybe I made a mistake. If you got it running, please open a pull request or also feel free to simply push your improvements.